tests/PackageHelpers.elm

```diff
@@ -117,7 +117,7 @@ packageHelpers =
                     cycles =
                         layout
                             |> Compiler.facts
-                            |> Result.map (List.any ((==) True) << detectCycles)
+                            |> Result.map (List.member True << detectCycles)
```

Comment:
Possible improvement
```diff
-detectCycles : List ( String, TypeAnnotation ) -> List Bool
+detectCycles : List ( String, TypeAnnotation ) -> List Bool
detectCycles list =
    let
        dict : Dict String TypeAnnotation
        dict =
            Dict.fromList list
    in
    dict
        |> Dict.keys
-       |> List.map
+       |> List.any
            (\key ->
                hasCycles dict key []
            )
```
This way `Result.map (List.member True << detectCycles)` can be simplified to `Result.map detectCycles`.

And this can be improved even further by avoiding `Dict.keys` by re-using `list` above (not entirely sure but I think?)

Reply:
Oh, let me go for it. That was the automatic fix but yeah, it can probably be improved

Reply:
Done!